_sass/_base.scss

```diff
@@ -172,11 +172,10 @@ table {
 $blockquote-color: $primary-text-color-dark;
 
 blockquote {
-  @include type-setting(2);
+  @include type-setting(1);
```

Comment:
We might also want to get rid of [L193-196](https://github.com/radicalbarracuda/sf-feminist-book-club/pull/23/files#diff-fd4aa8862bb3db5bc523e0b0e9b6084aR194) because of this 
![image](https://cloud.githubusercontent.com/assets/784889/25033903/111241d2-2097-11e7-9e33-abc516db65c0.png)